lib/perl/Genome/InstrumentData/AlignmentResult/Merged.pm

```diff
@@ -419,15 +419,14 @@ sub estimated_kb_usage {
         my $bam_size = $alignment->bam_size;
 
         unless (defined $bam_size) {
-            my @aln_bams = $alignment->alignment_bam_file_paths;
-            unless (@aln_bams) {
+            my $aln_bam = $alignment->get_bam_file;
+            unless ($aln_bam) {
                 die $self->error_message("alignment $alignment has no bams at " . $alignment->output_dir);
```

Comment:
This error message is not accurate now since get_bam_file would return either revivified bam in temp allocation or copied in-palce bam in temp allocation.